### ISSUE TITLE
Could instructions for Secrets subtract be clearer?

### ISSUE THREAD

**Issue by iHiD** [exercism/elixir#1170]
I may be wrong on this, but it caught out me, Brian and Erik, so I think I'm not.

---

The [example for subtract](https://github.com/exercism/elixir/blob/main/exercises/concept/secrets/.docs/instructions.md#2-create-a-subtractor) is:
```
subtractor = Secrets.secret_subtract(2)
subtractor.(3)
# => 1
```

And the [tests are](https://github.com/exercism/elixir/blob/main/exercises/concept/secrets/test/secrets_test.exs#L18-L37):
```
subtract = Secrets.secret_subtract(3)
subtract.(6) 
=> 3
```

However, the [instructions](https://github.com/exercism/elixir/blob/main/exercises/concept/secrets/.docs/instructions.md#2-create-a-subtractor) are: "It should return a function which takes one argument and subtracts from it the secret passed in to secret_subtract."

The functions above are subtracting the secret from the argument, not the argument from the secret. [My passing solution](https://exercism.org/tracks/elixir/exercises/secrets/solutions/iHiD) subtracts the secret from the argument too.

**Comment by iHiD** [comment]
Happy to PR a fix, but don't know which way round this should be changed.

Love the exercise though :)

**Comment by angelikatyborska** [comment]
I am pretty sure the current wording is correct, but if it trips up a native speaker, than maybe it needs to be improved somehow anyway.

The correct solution to this step is this:
```
  def secret_subtract(secret) do
    fn x -> x - secret end
  end
```

And the instructions are:

> It should return a function which takes one argument and subtracts from it the secret passed in to secret_subtract.

Let me rephrase the instructions adding more clarifications to the pronouns based on the example correct solution:

> The function `secret_subtract` should return a function which takes one argument `x` and subtracts from it (from `x`) the `secret` passed in to `secret_subtract`.

The detail that makes the biggest difference is the word order in: "subtracts from it", not to be confused with "subtracts it from" which would flip the meaning.

Does this make sense? How would you rephrase it to be more clear?

**Comment by iHiD** [comment]
Ooooh, sorry. Yes. "subtracts from it" not "subtracts it from". You're totally right that it's correct it it totally caught me out. "subtracts from it" isn't a phrase I feel I've heard before. I'm used to the thing being subtracted being in between "subtracts" and "from". (which may be less grammatically correct as it splits in the infinitive, but it sounds much more normal to me)

I think this would would have made sense to me:
> The function `secret_subtract` should return a function which takes one argument `x` and subtracts the `secret` (passed in to `secret_subtract`) from it.
